whenareyou/__init__.py

```diff
@@ -7,9 +7,10 @@
 from urllib.parse import quote_plus
 from pytz import timezone
 from tzwhere import tzwhere
+from django.conf.settings import GOOGLE_MAPS_API_KEY
```

Comment:
tying this to Django isn't a good idea.

Reply:
yeah this is supposed to be a simple library which isn't tied to django. Let's make this part of the signature?